pkg/proc/variables.go

```diff
@@ -645,7 +645,7 @@ func (scope *EvalScope) FunctionArguments(cfg LoadConfig) ([]*Variable, error) {
 		return nil, err
 	}
 	vars = filterVariables(vars, func(v *Variable) bool {
-		return (v.Flags & (VariableArgument | VariableReturnArgument)) != 0
+		return (v.Flags & VariableArgument) != 0
```

Comment:
We can't do this because of backwards compatibility: FunctionArguments + Locals == all local variables.

Reply:
I actually have a list of things I would like to change about the API, including this. I don't think they are compelling enough reasons to warrant a new API version. Goland took months to convert to APIv2, VSCode-go converted last week, after 2 years, and they are still using v1 by default. And the change from v1 to v2 was minimal. If we bump API we're looking at supporting 3 versions of the API for years to come.

The same effect here can be had by adding a single optional flag to ListFunctionArguments, it's not worth upgrading the API for this.

Reply:
I agree with this, let's wait for now.